activesupport/test/json/encoding_test.rb

```diff
@@ -422,6 +422,11 @@ def test_twz_to_json_when_wrapping_a_date_time
     assert_equal '"1999-12-31T19:00:00.000-05:00"', ActiveSupport::JSON.encode(time)
   end
 
+  def test_uri_to_json
+    uri = URI.parse('http://example.com')
+    assert_equal '"http://example.com"', ActiveSupport::JSON.encode(uri)
+  end
+
```

Comment:
Can you add it to https://github.com/rails/rails/blob/master/activesupport/test/json/encoding_test_cases.rb instead?